docs/cli/README.md

```diff
@@ -0,0 +1,107 @@
+# orkactl
+
+## Requirements
+
+For the v0 of orka, only some services are available, which you can see below:
+
+### Provided services
+
+- single tenant  
+- single node  
+- single workload kind (containers)  
+
+
+### Out of scope services (for v0)
+
+- Ressource limitation  
+- Multi tenant
+- Multi workload (VM, containers, bare metal)
+- Scalabilty
+- High Availability
+- Upgradable
+- Monitoring
+- Per tenant networks
+
+
+## High Level Architecture
+
+We are the CLI. As such, we will only communicate with the Cluster API and get information from it.
+
+![High Level Architecture document](https://media.discordapp.net/attachments/1119640478959280138/1122595576396120154/image.png)
```

Comment:
I suggest adding this Mermaid graph instead of referencing a Discord CDN link.

```mermaid
---
title: "Orka: High-level Architecture"
---
flowchart LR
    cli[CLI]

    subgraph Cluster
        subgraph Control Plane
            api[Cluster API]
            ctr[Controller]
            sch[Scheduler]
        end

        subgraph Nodes
            subgraph Node1
                a1[Agent]
            end
            subgraph Node2
                a2[Agent]
            end
            subgraph Node3
                a3[Agent]
            end
        end

    end

    cli <--> api
    api <--> ctr
    ctr <--> sch

    a1 & a2 & a3 --> sch

    a1 ~~~ a2 ~~~ a3
```